python/py-logilab-hmm/Portfile

```diff
@@ -21,17 +23,19 @@ livecheck.url       http://ftp.logilab.org/pub/hmm/
 livecheck.type      regex
 livecheck.regex     {logilab-hmm-(\d+(?:\.\d+)*)\.tar\.gz}
 
-checksums           md5     81f63768df022714e66f6d471245800a \
-                    sha1    6be2012ab3b7ed6ceac42b48a821f6e194c71f6f \
-                    rmd160  b62f3ac49e4d2681138e55490430c4bdfdf418f6
+checksums           rmd160  b62f3ac49e4d2681138e55490430c4bdfdf418f6 \
+                    sha256  62f51ea6c7b374cebdc3747b391d314336255cc48b0d2413722400f1d200a1ca \
+                    size    32856
 
-python.versions     26
+compilers.setup     require_fortran
+python.versions     27
 
-if {$subport ne $name} {
-    depends_lib         port:py${python.version}-logilab-common port:py${python.version}-numpy
+if {${subport} ne ${name}} {
+    depends_lib-append      port:py${python.version}-logilab-common \
+                            port:py${python.version}-numpy
 
-    universal_variant   no
-    python.add_archflags no
+    universal_variant       no
+    python.add_archflags    no
```

Comment:
Inconsistent whitespacing here

Reply:
what is inconsistent here - the whitespacing is multiples of four and it's aligned within the ```if {${subport} ne ${name}}```. What would you like to be changed?

Reply:
Sorry I shouldn't read codes when sleepy :/